### ISSUE TITLE
Installing modules from command line fails

### ISSUE THREAD

**Issue by ThyMYthOS** [dceejay/nodered.snap#11]
Install fails with `Unhandled rejection Error: EPERM: operation not permitted, chown '/root/snap/node-red/195/.npm/_cacache/content-v2/sha512/8e/ec'`

**Comment by ThyMYthOS** [comment]
My setup is a Raspi 3 with Ubuntu Snappy Core 16. I log in via ssh and use `sudo su` to get root. Then `cd /root/snap/node-red/current/` as written in the README. There I run `node-red.npm i node-red-contrib-smartfritz`.

**Comment by dceejay** [comment]
Hi - can you try it with 

    node-red.npm i --unsafe-perm node-red-contrib-smartfritz

**Comment by ThyMYthOS** [comment]
Same error as before.

**Comment by dceejay** [comment]
You can install via the manage palatte option inside Node-RED.

snappy core is even more tied down that snapd on vanilla Ubuntu - so will need to dig around to work out how best to fix command line.

**Comment by ThyMYthOS** [comment]
Yes, using `manage palette` in the web ui works as expected and installs packages to `/root/snap/node-red/current/node_modules`.

**Comment by dceejay** [comment]
Hmm - strangely on my fresh install  I install Node-RED snap - check it is running (via browser) then ssh in

    sudo su -
    cd ~/snap/node-red/current/
    node-red.npm i node-red-contrib-smartfritz
    snap restart node-red

And it works fine
